### PR TITLE
feat: add default prompt support

### PR DESCRIPTION
**Pull request by Ninzore**
Add a new option `-d` for adding a default prompt to the user prompt.

Allow users to generate images without input (in such cases they must toggle the `default` option).

Add a new shortcut `default` for quick accessing the default prompt without any user input.

### CODE REVIEW

**Comment by MaikoTan** [comment]
`basePrompt` doesn't solved this?

**Comment by Ninzore** [comment]
This function is designed to generate images without any user input, and when users want to input something, this one doesn't modify the input unless the users call the trigger.
On the other hand, the `basePrompt` modifies the user input, and the original implementation forces the user to input something, which is not very convenient in some use cases.
This `defaultPrompt` is intended to cooperate with the [dynamic prompt](https://github.com/adieyal/sd-dynamic-prompts), in order to generate random images with some pre-defined prompt. This function may also be extended in the future for more randomised word combinations.
However the name of it might be a little misleading, maybe consider another name?

File: src/locales/zh-CN.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ commands:
       strength: 图片修改幅度
       noise: 图片噪声强度
       hiresFix: 启用高分辨率修复
+      default: 添加默认提示词
```
Comment (MaikoTan) [inline src/locales/zh-CN.yml:25]: just 默认标签

File: src/locales/zh-TW.yml [hunk 2]
```diff
@@ -20,6 +20,7 @@ commands:
       strength: 圖像修改幅度
       noise: 圖像雜訊強度
       hiresFix: 啟用高分辨率修復
+      default: 默認提示詞
```
Comment (MaikoTan) [inline src/locales/zh-TW.yml:23]: No add translation in a PR, you could add it from crowdin.

**Review by MaikoTan — APPROVED** [review]
LGTM

**Pull request merged**